src/ConfigChanges/munge-util.js

```diff
@@ -23,10 +23,8 @@ exports.deep_add = function deep_add (obj, keys /* or key1, key2 .... */) {
         keys = Array.prototype.slice.call(arguments, 1);
     }
 
-    return exports.process_munge(obj, true/* createParents */, function (parentArray, k) {
-        var found = _.find(parentArray, function (element) {
-            return element.xml === k.xml;
-        });
+    return exports.process_munge(obj, true/* createParents */, (parentArray, k) => {
+        var found = _.find(parentArray, element => element.xml === k.xml);
```

Comment:
This would be slightly easier for me to understand at first glance:

```suggestion
        var found = _.find(parentArray, element => (element.xml === k.xml));
```